### PR TITLE
Update Pax baselines

### PR DESCRIPTION
**Pull request by ashors1**
Many of the existing Pax baselines are outdated. We have also introduced a number of new Pax tests (see #352) which currently lack baselines. This PR 

- Updates the (upstream and Rosetta) Pax baselines
- Loosens the loss threshold slightly to reduce the probability of false positives

This PR depends on #352.

### CODE REVIEW

**Comment by ashors1** [comment]
Note that due to the singleprocess failures mentioned in #352, there are currently no baselines for the 8DP singleprocess tests. Also because of these failures, the baseline for 2 FSDP + 4 TP + TE singleprocess test is based on only one test run.

File: .github/workflows/baselines/test_pax_mgmn_metrics.py [hunk 1]
```diff
@@ -24,7 +24,7 @@ def test_loss(baseline_filename):
     with open(baseline_filepath, "r") as baseline_file:
         end_step = json.load(baseline_file)["end_step"]
         loss_actual = test_utils.read_tb_tag(event_file, loss_summary_name)
-        assert loss_actual[end_step] < 1.6e-6, f"Loss at final step: {loss_actual[end_step]}, Expected loss < 1.6e-6"
+        assert loss_actual[end_step] < 1.75e-6, f"Loss at final step: {loss_actual[end_step]}, Expected loss < 1.75e-6"
```
Comment (terrykong) [inline .github/workflows/baselines/test_pax_mgmn_metrics.py:27]: What do you think about a check on both ends to guard against -inf/negative xent loss, both of which indicating something funny is going on?

```suggestion
        assert 0 <= loss_actual[end_step] < 1.75e-6, f"Loss at final step: {loss_actual[end_step]}, Expected 0 <= loss < 1.75e-6"
```

**Review by yhtang — COMMENTED** [review]
Similar to #352 , shall we wait until the existing issues got fixed before merging this?

**Comment by yhtang** [comment]
@ashors1 Shall we merge this PR now?

**Comment by ashors1** [comment]
I think this PR is safe to merge. @yhtang @terrykong can you approve?

**Review by terrykong — APPROVED** [review]

**Review by terrykong — APPROVED** [review]

**Comment by terrykong** [comment]
@yhtang @nouiz Just letting you know that this PR fixes the PAX baselines so that it surfaces the perf regression from 1/6. There's an internal bug tracking, but let us know today if you have any objections to merging.

Otherwise, @ashors1 can merge tomorrow

**Comment by yhtang** [comment]
No objection from my side. Please go ahead.

**Comment by nouiz** [comment]
No objection.

**Pull request merged**